client/state/sites/guided-transfer/test/actions.js

```diff
@@ -109,7 +114,9 @@ describe( 'actions', () => {
 		before( () => {
 			nock( 'https://public-api.wordpress.com:443' )
 				.post( `/wpcom/v2/sites/${sampleSiteId}/transfer` )
-				.times( 3 )
+				.times( 2 )
```

Comment:
Is `times` here dependent on the number of tests you have an number of times it's running on the endpoint? That seems cumbersome to maintain. Maybe instead vary the response by the sample site ID used. Here are a few example site IDs: `2916284`, `77203074`, `77203199`